lib/settings/settings.dart

```diff
@@ -0,0 +1,37 @@
+import 'package:flutter/foundation.dart';
+
+import 'persistence/local_storage_settings_persistence.dart';
+import 'persistence/settings_persistence.dart';
+
+class SettingsController {
+  final SettingsPersistence _store;
+
+  ValueNotifier<bool> soundsOn = ValueNotifier(false);
+  ValueNotifier<bool> musicOn = ValueNotifier(false);
```

Comment:
Would be really nice for inspiration use [AudioCubit](https://github.com/VeryGoodOpenSource/very_good_flame_game/blob/main/brick/__brick__/%7B%7Bproject_name.snakeCase()%7D%7D/lib/game/cubit/audio/audio_cubit.dart).
I don't want to say that proposed solution is not working, but it is not elegant.

Reply:
works as well, but better to have consistency through the code

Reply:
I don't see why this code is bad. It's copied from flutter casual game template